### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Dylan Langumier- Raphael Perret

### DISCUSSION THREAD

**Discussion by rp2709** [heig-vd-dai-course/heig-vd-dai-course#459] [Show and tell]
Une petite application de bataille navale multijoueur sera réalisée et permettera à deux clients de jouer l'un contre l'autre.
En option, si le temps le permet, les joueurs pourraient s'identifier et des statistiques conservées par joueur.

**Comment by GeraudSilvestri** [comment]
C'est bon pour nous

**Comment by maellemelody** [comment]
Lien de notre repo: https://github.com/Dylan-Langumier/DAI-Practical-Work-2

**Reply by maellemelody** [reply]
Et voici le [tag de tendu](https://github.com/Dylan-Langumier/DAI-Practical-Work-2/tree/Rendu)
